main.ts

```
@@ -317,8 +317,16 @@ async function run() {
   for (const scan of scan_info) {
     const { mobile_app_id, scan_id } = scan;
 
-    // Poll for scan completion with 30-second intervals
-    const maxWaitTime = 300000; // 5 minutes
+    var maxWaitTime = 300000; // 5 minutes
+    if (polling_timeout) {
+        maxWaitTime = parseInt(polling_timeout, 10);
+        // Fallback to default value if the value is incorrect
```

Please raise an error if the value is incorrect instead
If we silently revert to defaults, and it doesn't match their input the customer will think the plugin is just broken